samples/toolbarconfigurator/js/toolbarmodifier.js

```diff
@@ -291,7 +291,7 @@
 
 		AbstractToolbarModifier.prototype._createModifier.call( this );
 
-		this.modifyContainer.setHtml( this._toolbarConfigToListString() );
+		this.modifyContainer.setHtml( CKEDITOR.tools.htmlSafeByReview(this._toolbarConfigToListString(), 'Created using safe internal function that does not use user input.') );
```

Comment:
This is the only use of `_toolbarConfigToListString`. Maybe this function should return a Trusted Type (as in run this `htmlSafeByReview` right before the return statement of `_toolbarConfigToListString`).

Reply:
samples/

Reply:
LGTM